### ISSUE TITLE
Repeat experiments with a more complex problem

### ISSUE THREAD

**Issue by miguelesteras** [rdgreene/sa_tsp#23]

**Comment by miguelesteras** [comment]
@rdgreene Hi buddy, just thinking... for the more complex scenario, how did you plan to define end state? 

I would suggest we do as for the toy example and define it as a separate state with same rewards as start, and with the same coordinates. Which would mean adding this as an extra row to the coordinates .csv files.

Do you agree?

Miguel :dash:

**Comment by miguelesteras** [comment]
wow... that looks a lot like a fart :flushed:
Not quite what I intended.

Miguel :sparkles:

**Comment by rdgreene** [comment]
Yep, the loadTSPmatrix function is already set up to automatically define the end state for a graph of any size the same as we did for our toy example (in regards to initialising the R matrix anyway). In terms of plotting, I suggest we follow the same process as you did for the toy example (i.e. same coords for start and end). Does that make sense? 

Ronan 💖
